web/client/components/catalog/Catalog.jsx

```diff
@@ -360,7 +359,9 @@ class Catalog extends React.Component {
                             </Form>)}
                     footer={this.renderPagination()}>
                             <div>
-                                {this.renderResult()}
+                                { this.props.loading ? <div className="catalog-results loading"><Loader size="176" /><h4>
```

Comment:
i would keep the new loading stuff in the renderLoading function

Reply:
Initial, it was in dedicated function because it was used in multiple places but currently it is used in only one place. Besides the `Loader` by itself is already a component and the surrounding container make sense in that place. What do you think @MV88

Reply:
I know it is used only there, but i think is clearer to use a function if the condition of the ternary is true. it is just a matter of reability taste.

 { this.props.loading ? this.renderLoading() : this.renderResult() }
i think this is more readable then your solution. as I said is a matter of code style.

Also renderResult is used only once but we can decide to leave it there in order to shrink the render method